content/data-types/exercises/_index.md

```diff
@@ -0,0 +1,159 @@
+---
+title: "Exercises: Data Types"
+date: 2023-06-14T09:27:57-05:00
+draft: false
+weight: 3
+originalAuthor: Courtney Frey # to be set by page creator
+originalAuthorGitHub: Courtney Frey # to be set by page creator
+reviewer: Kimberly Horan # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+## Getting Started
+
+<!-- TODO: Link to chapter 1 for naming conventions-->
+
+Work on these exercises in the IntelliJ `java-web-dev-exercises` project. Create a new class for each numbered exercise. You may name the classes whatever you like, but use proper [Java Naming Conventions](Link to chapter 1) and make sure that the file name matches the class name.
```

Comment:
Suggest adding link to repo https://github.com/LaunchCodeEducation/java-web-dev-projects-17 and modify the wording to match name of repo.

Reply:
name has been updated - we are using the java-web-dev-projects repo